src/persistence.interfaces.ts

```diff
@@ -124,13 +123,9 @@ export interface IPersistence {
     decodePersistence(persistenceString: string): string;
     getCookieDomain(): string;
     getDomain(doc: string, locationHostname: string): string;
-    getAllUserAttributes(mpid: MPID): AllUserAttributes;
     getCartProducts(mpid: MPID): Product[];
     setCartProducts(allProducts: Product[]): void;
-    saveUserAttributesToPersistence(
-        mpid: MPID,
-        userAttributes: UserAttributes
-    ): void;
+    getUserIdentities(mpid: MPID): UserIdentities;
```

Comment:
it looks like this has been added back in after removing it from https://github.com/mParticle/mparticle-web-sdk/pull/869/files.  probably a rebasing issue.